app/org/views.py

```diff
@@ -19,6 +19,9 @@ def view_org(org_id):
     organization = Organization.query.filter_by(id=org_id).first()
     user = User.query.filter_by(id=organization.user_id).first()  
     pics = []
+    user = User.query.filter_by(id=organization.user_id).first()
```

Comment:
change logic to: if current_user = organization user (the user for this organization) , then flash
if user.approved is false, then redirect to error/error404